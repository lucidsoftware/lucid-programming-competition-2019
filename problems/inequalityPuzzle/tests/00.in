0000404010000000000000000
|<|>|||||<|||||||||||<|<|||||||||||<>|<|
