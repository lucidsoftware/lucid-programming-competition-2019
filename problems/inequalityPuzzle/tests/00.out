3125424513523411342545132